style.css

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+}
+
+#welcome {
+  font-size: 20px;
+  justify-content: end;
+}
+
+.menu {
+  align-self: center;
+  padding-left: 280px;
+}
+
+.headline_section {
+  position: relative;
+  text-align: center;
+}
+
+.text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.primary-text {
+  align-self: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  text-justify: inter-word;
+}
+
+#second {
+  color: #36b37e;
+}
+
+.second-text {
+  width: 327px;
+  height: 164px;
+  font-weight: 400;
+  font-size: 20px;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- Kindly ensure that the color, font size, line height, and font weight of the introductory text match the Figma design you chose. You can check out the Figma design specification for the exact properties used👍🏽

## Your design

![image](https://user-images.githubusercontent.com/95297251/191822735-8c03f06a-9c3b-4f29-83b0-0558a6aa00c3.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/191823707-d6898faf-a3e7-41bc-ac82-c192427198fe.png)

Reply:
Designed as per Figma design. Please check.